src/locales/sv.ts

```diff
@@ -109,4 +109,39 @@ export default {
       <p>D-Dagens projektgrupp</p>
     `,
   },
+  faq: {
+    box1: "OM D-DAGEN",
+    box2: "MARKNADSFÖRING",
+    box3: "ALLMÄNT",
+    box4: "EVENEMANG",
+    table1row1:"VAD ÄR D-DAGEN?",
+    table1row2: "NÄR ÄR ÅRETS D-DAGEN?",
+    table1row3: "VAR ÄR D-DAGEN?",
+    table1text1: "D-Dagen är den årliga karriärmässan som anordnas av Datasektionen på Kungliga Tekniska Högskolan. D-Dagen är numera Nordens största karriärmässa inom IT.",
+    table1text2: "Årets D-Dagen äger rum torsdag den 12:e oktober.",
+    table1text3: "D-Dagen hålls i vårt kårhus Nymble, som ligger på KTH.",
+    table2row1: "KAN VÅRT FÖRETAG FÅ MER EXPONERING MOT STUDENTER?",
+    table2row2: "KAN VÅRT FÖRETAG MARKNADSFÖRA JOBBERBJUDANDEN OCH LIKNANDE VIA D-DAGEN?",
+    table2text1_1: "Absolut, vi har flera olika paket för att ni ska kunna nå ut till fler studenter, kontakta",
+    table2text1_2: "för mer info.",
+    table2text2: "Ja, det primära syftet med D-Dagen är att marknadsföra karriärmöjligheter som jobberbjudanden, traineeprogram, sommarpraktikplatser och liknande till våra studenter.",
+    table3row1: "HUR ANMÄLER VI OSS TILL MÄSSAN?",
+    table3row2: "VILKEN ÄR SISTA DATUMET FÖR ATT GÖRA EN INTRESSEANMÄLAN?",
```

Comment:
I think the phrasing here is a bit strange. It might be worth changing.

Reply:
Do you mean it should be "VILKET" instead of "VILKEN"

Reply:
Yes, that is a part of it, but still not a completely clear sentence imo. Not super important though